config/org.jenkinsci.plugins.configfiles.GlobalConfigFiles.xml

```diff
@@ -18,6 +18,78 @@ node {
       <providerId>org.jenkinsci.plugins.configfiles.groovy.GroovyScript</providerId>
     </org.jenkinsci.plugins.configfiles.groovy.GroovyScript>
     <org.jenkinsci.plugins.configfiles.groovy.GroovyScript>
+      <id>go-ipfs-jenkinsfile</id>
+      <name>go-ipfs-jenkinsfile</name>
```

Comment:
sweet, so now go-ipfs have a `ci/jenkins` file with `go-ipfs-jenkinsfile` in it?